reference/api/faceting.md

```diff
@@ -0,0 +1,98 @@
+# Faceting
+
+_Child route of the [settings route](/reference/api/settings.md)._
+
+This route allows you to configure the faceting settings for an index.
+
+Faceting settings can also be updated directly through the [global settings route](/reference/api/settings.md#update-settings) along with the other settings.
+
+To learn more about filtering and faceting, refer to our [dedicated guide](/learn/advanced/filtering_and_faceted_search.md).
+
+::: warning
+Updating the settings means overwriting the default settings of Meilisearch. You can reset to default values using the `DELETE` routes.
+:::
+
+## Get faceting settings
+
+<RouteHighlighter method="GET" route="/indexes/{index_uid}/settings/faceting"/>
+
+Get the faceting settings of an index. The index [`uid`](/learn/core_concepts/indexes.md#index-uid) is required.
+
+### Example
+
+<CodeSamples id="get_faceting_settings_1" />
+
+#### Response: `200 OK`
+
+```json
+{
+  "maxValuesPerFacet": 100
+}
+```
+
+### Returned fields
+
+#### `maxValuesPerFacet`
+
+Maximum number of facet values returned for each facet.
+
+## Update faceting settings
+
+<RouteHighlighter method="PATCH" route="/indexes/{index_uid}/settings/faceting"/>
+
+Partially update the faceting settings for an index. The index [`uid`](/learn/core_concepts/indexes.md#index-uid) is required.
+
+### Body
+
+#### `maxValuesPerFacet`
+
+**Type:** integer
+**Default value:** `100`
+
+Configure the maximum number of facet values returned for each facet. Values are alphabetically sorted.
```

Comment:
I am not sure that we should specify that the results are returned in this order, as it is not a specified behavior, that's just how the engine works right now and not how the specs want it.

Reply:
I think the spec might have been recently updated: https://github.com/meilisearch/specifications/pull/157/files#diff-6738a194cb97b82d2f0051b59a87cb03a343c5bad665ee11c2da0faf00a273c8R35

Unless I'm misunderstanding what we mean by lexicographical order on L#35?

Reply:
We mean that if there were capital letters, for example, the capital one would be before the lowercase versions. It really applies to the raw values encoded as UTF-8. The real order is the order in terms of byte representation and not determined by human things like alphabetic or numeric order. For example `2` will be listed before `1000`.

Reply:
Indeed, the actual behavior has been updated in the specification because I had internal questions about it.

I think it's not bad to explain it, it also allows people to know about it and to tell us if there are frustrations about it so we can improve it later. (my 2 cents)

Reply:
Thanks for the clarification. I was hoping we could have gotten away with alphabetical order since it's a bit more user-friendly, but, indeed, it sacrifices too much technical accuracy. 

I have accepted @gmourier's suggestion changing "alphabetical" to "ascending lexicographical" as I also think some users might appreciate knowing the expected behaviour.

Thanks to both of you for the assistance!